readthedocs/projects/tasks/search.py

```diff
@@ -136,6 +166,22 @@ def _get_indexers(*, version, search_ranking, search_ignore, search_index_name=N
             search_index_name=search_index_name,
         )
         indexers.append(search_indexer)
+
+    # File tree diff is under a feature flag for now,
+    # and we only allow to compare PR previous against the latest version.
+    has_feature = version.project.has_feature(
+        Feature.GENERATE_MANIFEST_FOR_FILE_TREE_DIFF
+    )
+    create_manifest = has_feature and (
+        version.is_external or version == version.slug == LATEST
```

Comment:
I think we should likely check for the Project's default version, not latest here.

Reply:
Do you mean the version `/` redirects to? That can be the stable version, when doing a PR you usually do it against main/master, which is usually what latest points to.

Reply:
I think what @stsewd says is fine for now. This should eventually be configurable, tho.

Reply:
BTW, it this correct? `or version == version.slug == LATEST`? shouldn't be `and version and version.slug == LATEST`?